sciter/event.py

```diff
@@ -186,6 +189,9 @@ def _on_script_call(self, f):
             except Exception as e:
                 import traceback
                 traceback.print_exc()
+                if self.script_call_exception_handler:
+                    # if exception handler is defined, call it with (func_name, exception) as arguments
+                    self.script_call_exception_handler(fname, e)
                 rv = e
```

Comment:
@AlecHaring you see this? We return the exception as an error back to the script. Do you think it will make sense allowing to override the returned exception? Perhaps, optionally. 

Like, `rv = self.script_call_exception_handler(fname, e)`

Reply:
Hmm yeah, I like that idea. It would allow for more control over how the exception is handled. Right now, the exception handler can only be used to get notified when an exception has occurred rather than actually doing something with the exception